### ISSUE TITLE
rename panel to '3Di panel'

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-api-qgis-client#72]
* threedi-api-qgis-client version: 1.5.0.b
* Python version:
* Operating System:

### Description

the panel is qgis is called 'panel' makes it hard for users to find it 
please rename to 3Di panel

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by Jonasvsl** [comment]
this panle: 
![image](https://user-images.githubusercontent.com/1453159/86108865-fe88f900-bac3-11ea-921b-b7541f51c02f.png)

![image](https://user-images.githubusercontent.com/1453159/86108914-0d6fab80-bac4-11ea-9fed-52d52cf2c23c.png)

**Comment by DamnBack** [comment]
Fixed in the version 2.0.0
